app/views/applications/deploy.html.erb

```diff
@@ -56,11 +56,19 @@
         <% if @staging_dashboard_url %>
           <p>
             <span class="glyphicon glyphicon-stats" aria-hidden="true"></span>
-            <a target="_blank" href="<%= @staging_dashboard_url %>">Staging dashboard</a>:
+            <a target="_blank" href="<%= @staging_dashboard_url %>">Carrenza Staging dashboard</a>:
             monitor your deployment to check that it doesn't cause any problems.
           </p>
+        <% elsif @aws_staging_dashboard_url %>
```

Comment:
The page displays two buttons, one for Carrenza Staging and one for AWS Staging, so this `elsif` makes no sense. 😳

Reply:
Fixed. There are now two `if` blocks. I did think about doing `if @staging_dashboard_url && @aws_staging_dashboard_url`, but if one of them doesn't exist then the other won't display at all, which isn't ideal.